ddtrace/llmobs/_llmobs.py

```diff
@@ -239,15 +239,19 @@ def export_span(cls, span: Optional[Span] = None) -> Optional[ExportedLLMObsSpan
             except (TypeError, AttributeError):
                 log.warning("Failed to export span. Span must be a valid Span object.")
                 return None
-        span = cls._instance.tracer.current_span()
+        span = cls._instance.current_span()
         if span is None:
             log.warning("No span provided and no active LLMObs-generated span found.")
             return None
-        if span.span_type != SpanTypes.LLM:
-            log.warning("Span must be an LLMObs-generated span.")
-            return None
         return ExportedLLMObsSpan(span_id=str(span.span_id), trace_id="{:x}".format(span.trace_id))
 
+    def current_span(self) -> Optional[Span]:
+        """Returns the current active LLMObs span."""
+        if self.enabled is False:
+            return None
```

Comment:
if we don't require `cls.enabled` to be true to start spans should we remove this check here as well? Since we've run into issues with customer code paths breaking when llmobs is turned off when customers are running unit tests for example

Reply:
Fair point, will remove that check.